wiki/Tournaments/ATT/2021/en.md

```diff
@@ -0,0 +1,143 @@
+  - [Goose house - Hikaru nara (radar) \[Solo!\]](https://osu.ppy.sh/beatmapsets/1207466#taiko/2514437)
+- FreeMod
+  - [A? - Arrogation (Cut Ver.) (ZTH) \[Oni\]](https://osu.ppy.sh/beatmapsets/1291023#taiko/2681860)
+  - [t+pazolite - Our Wrenally (Uncut Edition) (komasy) \[Oni\]](https://osu.ppy.sh/beatmapsets/892110#taiko/1864987)
+- Tiebreaker
+  - **[Venetian Snares - Epidermis (Horiiizon) \[HiroK's Oni\]](https://osu.ppy.sh/beatmapsets/1262133#taiko/2663673)**
+
+## Ruleset
+
+### General rules
+
+1. The tournament is open to players living in Australia, New Zealand, Malaysia, Singapore, Indonesia, Philippines, and Thailand. Recommended rank is #7,500+.
+2. The tournament style is a double-elimination bracket.
+3. Players will be seeded by global rank.
+4. The tournament will take the best of the first 32 entrants (given they fit the next rule).
+5. There is an ambiguous rank limit, as many countries have a large gap somewhere in their top five players, which leaves a large gap in competitive skill. As a result, some players may be denied entry to participate (as a guideline, around 8,500 pp is the desired cap).
+6. Poor attitude towards other players or tournament staff will result in disqualification (this point applies to staff members as well).
+
+### Match rules
+
+1. The tournament is a 1v1 osu!taiko match; highest score wins.
+2. Scoring will be calculated via **ScoreV2**.
+3. The mappool will change in size depending on round, but at a very minimum will contain 3 NoMod, 2 Hidden, 2 HardRock, 2 DoubleTime, 2 FreeMod, and 1 Tiebreaker.
+   - FreeMod consists of Hidden, HardRock, or Hidden+HardRock. No other mods/combinations of mods may be used (this includes NM).
+4. Players can request to use Nightcore instead of DoubleTime, but the opponent must also agree upon the request.
+5. Players are allowed one ban per match.
+6. Players can pick a warmup with the max length of **four minutes**.
+7. `!roll` will determine who will pick the first beatmap.
+8. The player with the lowest `!roll` score will ban first, followed by their opponent.
+9. Players have 10 minutes in Group Stage or 15 minutes in every stage above to attend their match before a win by default is given to the other player.
+10. If both players fail, the player with the highest failed score wins the point.
+11. If the scores are tied, then the beatmap will be played again and no point(s) will be awarded.
+    - If there exists two ties in a row on the same beatmap, the map may be nullified and not reused. The player whom selected the tied map may pick again.
+
+#### Tournament structure
+
+1. Seeding is based off of the player's rankings; there will be four seed categories and every player will fit into one of these depending on their total Taiko pp compared to the rest of the players. Groups for the Group Stage will be groups of four, where each player is from a different seed category. (This may change depending on the number of participants.)
```

Comment:
```suggestion
1. Seeding is based off of the player's rankings; there will be four seed categories and every player will fit into one of these depending on their total osu!taiko pp compared to the rest of the players. Groups for the Group Stage will be groups of four, where each player is from a different seed category. (This may change depending on the number of participants.)
```